scheduler/scheduler-server/src/test/java/functionaltests/db/schedulerdb/TestLoadJobsPagination.java

```diff
@@ -65,6 +65,11 @@ private TaskFlowJob createJob(String name, JobPriority priority, String projectN
         } else {
             job.setProjectName(this.getClass().getSimpleName() + " project");
         }
+        if (bucketName != null) {
+            job.setBucketName(bucketName);
+        } else {
```

Comment:
why do you **have to** set a non-null bucket name if not provided in the parameter in this test ?